cypress/utils/dashboards/vis-augmenter/helpers.js

```diff
@@ -320,3 +320,35 @@ export const filterByObjectType = (type) => {
     .click({ force: true });
   cy.wait(3000);
```

Comment:
not related to this PR but you guys have a lot of `wait(time in ms)` statements all across your tests. This is almost always unnecessary and can save you guys a lot of time in your tests.